Hardware/Hardware/Printer/AirPrintReceipt/AirPrintReceiptPrinterService.swift

```diff
@@ -0,0 +1,28 @@
+import UIKit
+
+/// Barebones Implementation of the ReceiptPrinterService that integrates with AirPrint
+/// Will be iterated in https://github.com/woocommerce/woocommerce-ios/issues/3982
```

Comment:
This exists, at this point, pretty much so that we can actually test that the receipt is being generated. The proper implementation will come in a later PR

Reply:
Do we need a ReceiptPrinterService or just a PrinterService - and could that be shared with shipping label printing?

We could define a renderer protocol, implement it for receipts (and labels, and whatever in the future like inventory bar codes) and pass the renderer in as a dependency to a generic PrinterService

Reply:
I also considered that, but I thought it would be better to not reach for more than what we need for the beta at this point. We can always rename this service later if we add support for Shipping Labels.

Reply:
That being said I'll rename to PrinterService right away